roles/aap_ocp_install/tasks/install-platform.yml

```diff
@@ -0,0 +1,172 @@
+    validate_certs: "{{ aap_ocp_install_connection['validate_certs'] | default(omit) }}"
+    kind: Route
+    name: "{{ aap_ocp_install_platform['instance_name'] | mandatory }}-eda"
+    api_version: route.openshift.io/v1
+    namespace: "{{ aap_ocp_install_platform['namespace'] | default(aap_ocp_install_namespace) | mandatory }}"
+  when:
+    - aap_ocp_install_eda is defined
+  register: __aap_ocp_install_eda_route_result
+  until: __aap_ocp_install_eda_route_result['resources']
+  retries: 60  # Wait for 15 minutes (60*15/60)
+  delay: 15
+
+- name: Store automation eda route
+  ansible.builtin.set_fact:
+    __aap_ocp_install_eda_route: "{{ __aap_ocp_install_eda_route_result['resources'][0]['status']['ingress'][0]['host'] }}"
+  when:
+    - aap_ocp_install_eda is defined
+
+- name: Ensure automation eda API is available
+  ansible.builtin.uri:
+    url: "https://{{ __aap_ocp_install_eda_route }}/api"
+    validate_certs: "{{ aap_ocp_install_connection['validate_certs'] | default(omit) }}"
+    method: GET
+    status_code:
+      - 200
+  when:
+    - aap_ocp_install_eda is defined
+  register: _aap_ocp_install_eda_available
+  until: (_aap_ocp_install_eda_available['status'] == 200) and ('migrations_notran' not in _aap_ocp_install_eda_available['url'])
+  retries: 120  # Wait for 30 minutes (120*15/60)
+  delay: 15
+
+- name: Wait for operator to create the automation hub route
+  kubernetes.core.k8s_info:
+    host: "{{ __aap_ocp_install_auth_results['openshift_auth']['host'] }}"
+    api_key: "{{ __aap_ocp_install_auth_results['openshift_auth']['api_key'] }}"
+    validate_certs: "{{ aap_ocp_install_connection['validate_certs'] | default(omit) }}"
+    kind: Route
+    name: "{{ aap_ocp_install_platform['instance_name'] | mandatory }}-hub"
+    api_version: route.openshift.io/v1
+    namespace: "{{ aap_ocp_install_platform['namespace'] | default(aap_ocp_install_namespace) | mandatory }}"
+  when:
+    - aap_ocp_install_hub is defined
+  register: __aap_ocp_install_hub_route_result
+  until: __aap_ocp_install_hub_route_result['resources']
+  retries: 60  # Wait for 15 minutes (60*15/60)
+  delay: 15
+
+- name: Store automation hub route
+  ansible.builtin.set_fact:
+    __aap_ocp_install_hub_route: "{{ __aap_ocp_install_hub_route_result['resources'][0]['status']['ingress'][0]['host'] }}"
+  when:
+    - aap_ocp_install_hub is defined
+
+- name: Ensure automation hub API is available
+  ansible.builtin.uri:
+    url: "https://{{ __aap_ocp_install_hub_route }}/api"
+    validate_certs: "{{ aap_ocp_install_connection['validate_certs'] | default(omit) }}"
+    method: GET
+    status_code:
+      - 200
+  when:
+    - aap_ocp_install_hub is defined
+  register: _aap_ocp_install_hub_available
+  until: (_aap_ocp_install_hub_available['status'] == 200) and ('migrations_notran' not in _aap_ocp_install_hub_available['url'])
+  retries: 120  # Wait for 30 minutes (120*15/60)
+  delay: 15
```

Comment:
This check is failing for me. Doing a get of the Hub route with `/api` returns a `status_code` of 404. However, using a path of `/api/galaxy/pulp/api/v3/` returns a 200 `status_code` and allows the task to complete.